src/styles/getMuiTheme.js

```diff
@@ -4,6 +4,21 @@ import ColorManipulator from '../utils/color-manipulator';
 import autoPrefix from './auto-prefix';
 import lightBaseTheme from './baseThemes/lightBaseTheme';
 import zIndex from './zIndex';
+import {autoprefixer, rtl} from './transformers';
+import compose from 'lodash.flowright';
+import warning from 'warning';
+
+const CALLED_ONCE = Symbol('prepareStyles/calledOnce');
+
+const callOnce = (transformer) => (style) => {
```

Comment:
Can we avoid to call this function in production with `if (process.env.NODE_ENV !== 'production') {`?